src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.csproj

```diff
@@ -7,7 +7,9 @@
     <AssemblyTitle>NuGet Command Line</AssemblyTitle>
     <PackageId>NuGet.CommandLine</PackageId>
     <TargetFramework>$(NETFXTargetFramework)</TargetFramework>
-    <GenerateDocumentationFile>false</GenerateDocumentationFile>
+    <GenerateDocumentationFile Condition="'$(IsOfficialBuild)' == 'true'">false</GenerateDocumentationFile>
+    <GenerateDocumentationFile Condition="'$(IsOfficialBuild)' != 'true'">true</GenerateDocumentationFile>
+    <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
```

Comment:
Isn't this globally set already?

https://github.com/NuGet/NuGet.Client/blob/9b1ed6bf04ebc23342a69c283c1fb98bb219524d/build/common.project.props#L127

Reply:
I read that as saying, if $(EnforceCodeStyleInBuild) is set to anything other than empty string, then it sets the property to true. So if we don't set it somewhere it won't put any value, right?

Reply:
It effectively sets a default. 
If not set, it'll set true, and since we don't set it, it's gonna be true. 

It's a common msbuild pattern, because this allows me to write msbuild -t:build -p:EnforceCodeStyleInBuild=false from the commandline and it'll be respected.

Reply:
Shouldn't it be 
```xml 
<EnforceCodeStyleInBuild Condition=" '$(EnforceCodeStyleInBuild)' == '' " >true</EnforceCodeStyleInBuild> 
```

For setting it default to true if nothing is set?

Reply:
See my brain abstracted that detail.
You're absolutely correct :D 

We should absolutely start by enable that everywhere. 

Locally on my machine, if I set that to true, I only get 2 different errors. 

That could be a good start, as we ensure that EnforceCodeStyleInBuild is actually respected everywhere.

Reply:
So I did that as a separate PR: https://github.com/NuGet/NuGet.Client/pull/5809